### PR TITLE
fix numpy + regex warnings

### PR DESCRIPTION
**Pull request by alexlancaster**
@rsingle I noticed that the calls to `math.sqrt` were producing `numpy` warnings.  I think this is due to being passed an `ndarray` rather than a scalar, so I modified this to take the first element of the `F_2_1_prime` or `F_1_2_prime`, but I'm not sure if this is generalizable solution.  Not sure why those variables are arrays in the first place.

### CODE REVIEW

**Comment by alexlancaster** [comment]
@rsingle does this look good to merge?  can you do a quick review?

File: src/PyPop/Haplo.py [hunk 1]
```diff
@@ -752,11 +752,11 @@ def _compute_LD(haplos, freqs, compute_ALD=False, debug=False):
         F_2 = 0.0
         F_1_2 = 0.0
         for i in numpy.unique(alleles1):
-           af_1 = numpy.unique(a_freq1[alleles1==i])
+           af_1 = numpy.unique(a_freq1[alleles1==i])[0]  # take the first element of ndarray
```
Comment (rsingle) [inline src/PyPop/Haplo.py:755]:
```suggestion
           af_1 = numpy.unique(a_freq1[alleles1==i])[0]  # take the first element of ndarray (default behaviour)
```

**Review by rsingle — CHANGES_REQUESTED** [review]
Looks good

**Review by rsingle — APPROVED** [review]
All set

**Pull request merged**